combat-trainer.lic

```diff
@@ -1816,6 +1819,12 @@ class TrainerProcess
       pickbox(game_state)
     when 'Hunt'
       bput('hunt', 'You take note of ', 'You find yourself unable to hunt in this area') unless game_state.retreating?
+    when 'Teach'
```

Comment:
❓ Is my understanding correct that the 'exp timer' value for 'Teach' is 'how often to check the room for students'?

Reply:
yeah, didn't want to make this too smart here. I don't care if I miss a teaching pulse while hiding (which stops the class) enough to juggle interrupting more important things like combat if it drops off, perfectly happy to just check to teach every minute or two.

Reply:
:+1: